collector/metrics/metric_collector.go

```diff
@@ -53,8 +53,7 @@ var GaugeMetricsMap map[string]prometheus.GaugeVec
 var CounterMetricsMap map[string]prometheus.CounterVec
 var SummaryMetricsMap map[string]prometheus.Summary
 
-// DataSource 0 meta server, 1 replica server.
-var DataSource int
+// RoleByDataSource 0 meta server, 1 replica server.
```

Comment:
It‘s corresponding to the variables in line 36, it would be better to mention the variable names instead of the magic numbers (0,1) again.